master/reference/calicoctl/resources/ippool.md

```diff
@@ -34,7 +34,7 @@ spec:
 
 | Field       | Description                 | Accepted Values   | Schema | Default    |
 |-------------|-----------------------------|-------------------|--------|------------|
-| cidr     | IP range to use for this pool.  | A valid IPv4 or IPv6 CIDR. | string | |
+| cidr     | IP range to use for this pool.  | A valid IPv4 or IPv6 CIDR. Must be at least a `/26` CIDR for IPv4 and `/122` CIDR for IPv6. Must not overlap with the Link Local range `169.254.0.0/16` or `fe80::/10`. | string | |
```

Comment:
Are there any other things that the pool mustn't overlap with?  Maybe Host subnet, cluster services subnet, etc?  If you don't want to list them all, maybe hint at them?

Reply:
Those are not hard requirements, and we don't validate against those since we don't have a way of tracking those during creation time. The ones I've listed are validated during creation time

Reply:
"at least /26" ... is open to interpretation, how about an explicit range, e.g. /26-/3?). otherwise looks fine.

Reply:
@bcreane the reason I didn't specify the upper limit ie because we haven't tested it to make sure if our IPAM can handle that many IPs, it's a theoretical limit, and putting it here makes it official (suggesting it's tested). 
However I agree "at least /26" doesn't sound that good, after some thinking, I still can't come up with a better sentence. I'm open to any other suggestions